lib/signaling/v2/iceconnectionmonitor.js

```diff
@@ -35,25 +35,47 @@ class IceConnectionMonitor {
       _timer: {
         value: null,
         writable: true,
+      },
+      _wasInactive: {
```

Comment:
@makarandp0 Nitpick: can we rename this as `isInactive`? "Was" implies its no longer inactive.